components/SearchResult.vue

```diff
@@ -10,12 +10,10 @@
       >
         {{ result.object_name }}
       </nuxt-link>
-      <source-tag :text="result.document.key" :title="result.document.name" />
-      <span
-        class="font-sans text-sm uppercase text-granite before:mx-2 before:content-['|']"
-      >
+      <span class="ml-2 font-sans text-sm uppercase text-granite">
         {{ result.object_model.match(/[A-Z][a-z]+/g).join(' ') }}
       </span>
+      <source-tag :text="result.document.key" :title="result.document.name" />
```

Comment:
Re-ordered so elements go from large & most-important (type) to small & least-important (source tag)